src/Fieldtypes/TemplateFolder.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Statamic\Fieldtypes;
+
+use Statamic\Facades\Folder;
+
+class TemplateFolder extends Relationship
+{
+    protected $component = 'template_folder';
+
+    protected function toItemArray($id, $site = null)
+    {
+        return ['title' => $id, 'id' => $id];
+    }
+
+    public function getIndexItems($request)
+    {
+        return Folder::disk('resources')
+            ->getFoldersRecursively('views')
+            ->map(function ($folder) {
+                $folder = str_replace_first('views/', '', $folder);
+
+                return ['id' => $folder, 'title' => $folder];
+            })
+            ->prepend(['id' => '/', 'title' => '/'])
```

Comment:
Was there a reason to have `/` pushed onto this list?

If you choose `/` then no templates appear in the list.

![CleanShot 2022-06-30 at 15 34 38](https://user-images.githubusercontent.com/105211/176762411-15140709-12b9-4fd3-9f80-3f1f93839070.png)

I'd rather if you don't want to select a folder, you just remove your selection instead of choosing `/`.

Reply:
🤔 Yeah that is unnecessary. I think it's just because I duplicated `AssetFolder.php`. Thanks for sorting!